tests/postgres/vm.test.ts

```diff
@@ -76,6 +81,52 @@ test('setupPlugin async', async () => {
     expect(newEvent.event).toEqual('haha')
 })
 
+test('teardownPlugin sync', async () => {
```

Comment:
It's not clear what this test is testing compared to the other one. When will this fail and the other one not?

Reply:
Indeed, removed one of them.